modules/java_api/cpp/input_info.cpp

```diff
@@ -132,3 +132,5 @@ JNIEXPORT jlong JNICALL Java_org_intel_openvino_InputInfo_GetTensorDesc(JNIEnv *
     }
     return 0;
 }
+
+JNIEXPORT void JNICALL Java_org_intel_openvino_InputInfo_delete(JNIEnv *env, jobject obj, jlong addr) {}
```

Comment:
please add comments that we do not call delete native object because we do not own them (no new has been called)